tests/integration/loginlogout/cmd_login_logout_test.go

```diff
@@ -39,29 +39,27 @@ var _ = Describe("odo login and logout command tests", func() {
 		})
 	})
 
-	Context("Run login tests with no active projects, having default is also considered as not having active project", func() {
-		It("Should login successfully with username and password without any projects with appropriate message", func() {
+	Context("when running login tests", func() {
+		It("should successful with correct credentials and fails with incorrect token", func() {
 			currentUserToken1 = oc.GetToken()
+			// Login successful without any projects with appropriate message
 			session1 = helper.CmdShouldPass("odo", "login", "-u", loginTestUserForNoProject, "-p", loginTestUserPassword)
 			Expect(session1).To(ContainSubstring("Login successful"))
 			Expect(session1).To(ContainSubstring("You don't have any projects. You can try to create a new project, by running"))
 			Expect(session1).To(ContainSubstring("odo project create <project-name>"))
 			session1 = oc.GetLoginUser()
 			Expect(session1).To(ContainSubstring(loginTestUserForNoProject))
-			// One initialization needs one login, hence it happens here
-			testUserToken1 = oc.GetToken()
-		})
 
-		It("Should login successfully with token without any projects with appropriate message", func() {
+			// Login successful with token without any projects with appropriate message
+			testUserToken1 = oc.GetToken()
```

Comment:
why are we getting the token twice?

Reply:
@mik-dass The variable name itself itself suggest that this token is for test user not for the first current user ```developer```.

testUserToken1 = oc.GetToken() gets the login token for [user](https://github.com/openshift/odo/blob/master/tests/integration/loginlogout/cmd_login_logout_test.go#L13) ```loginTestUserForNoProject```. 

currentUserToken1 = oc.GetToken() gets the login token for user ```developer```. It is the same ```developer``` user when we do the login for first time CI test run. For reference check https://github.com/openshift/odo/blob/master/.travis.yml#L35 and https://github.com/openshift/odo/blob/master/scripts/configure-installer-tests-cluster.sh#L86

Reply:
@mik-dass I have added comments. PTAL